.quickstart/quickstart.yml

```diff
@@ -17,6 +17,10 @@ multi_connector_config:
     database_key: twitter_organic_database
     union_schema_key: twitter_organic_union_schemas
     model_variable: social_media_rollup__twitter_enabled
+  youtube_analytics:
+    database_key: youtube_analytics_database
+    union_schema_key: youtube_analytics_union_schemas
+    model_variable: social_media_rollup__youtube_enabled
 
 table_variables: {}
```

Comment:
I think we may need to add the `table_variables` from the `youtube_analytics` [quickstart.yml](https://github.com/fivetran/dbt_youtube_analytics/blob/main/.quickstart/quickstart.yml#L6-L8) -- in the Ad Reporting package [quickstart.yml](https://github.com/fivetran/dbt_ad_reporting/blob/main/.quickstart/quickstart.yml#L49-L58), we have the individual package table variables. 

Looks like the only other Social Media individual package with a table_variable config is [Twitter](https://github.com/fivetran/dbt_twitter/blob/main/.quickstart/quickstart.yml#L6-L8) -- we might need to add this as well?

Reply:
thanks for catching!

Reply:
and added this to changelog notes